### PR TITLE
Field's Option constructor class made public.

### PR DESCRIPTION
**Pull request by russellhoff**
I've made Option's constructor public.

### CODE REVIEW

**Comment by russellhoff** [comment]
please, could you accept this pull request?

**Comment by russellhoff** [comment]
What happened to the pull request?

**Comment by russellhoff** [comment]
@stephenc something happened to jenkins...

**Comment by russellhoff** [comment]
Hello @stephenc , could you tell me how to improve the code to be acceptable by jenkins?

**Comment by stephenc** [comment]
@aheritier do you know where that jenkins is. I cannot access even when on vpn

**Comment by aheritier** [comment]
strange @stephenc I have no idea
no idea @Dohbedoh ?
The one I use is https://support.ci.cloudbees.com/job/DEV/job/zendesk-java-client/

**Comment by russellhoff** [comment]
Hello? Anything to say about that PR?

**Comment by russellhoff** [comment]
But Jenkins said "All checks have failed"

**Comment by russellhoff** [comment]
I'm waiting for any response....

**Comment by russellhoff** [comment]
Hi there! What happened here? Have all of you been abducted by an alien? :p

**Comment by gvh1234** [comment]
I believe this can be merged? It's causing pain

File: src/main/java/org/zendesk/client/v2/model/Field.java [hunk 1]
```diff
@@ -265,10 +265,10 @@ public String toString() {
         private String name;
         private String value;
 
-        private Option() {
+        public Option() {
```
Comment (johnou) [inline src/main/java/org/zendesk/client/v2/model/Field.java]: Why would you need to create an option with uninitialised values? This should only be required for Jackson and should be ok to leave as private.

Comment (russellhoff) [inline src/main/java/org/zendesk/client/v2/model/Field.java]: Is there other way to construct Options? You tell me, please. I was developing a project which needed the construction of Options.

Comment (johnou) [inline src/main/java/org/zendesk/client/v2/model/Field.java]: Yes, with the second constructor which takes the name and value args.

**Pull request merged**